Assets/Scripts/PlayerCamera.cs

```diff
@@ -4,17 +4,34 @@
 
 public class PlayerCamera : MonoBehaviour
```

Comment:
Now this class do 2 things:
Moving camera and animate bob move. It would be nice to move bob logic to new script